<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="gb_screen">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>261</width>
     <height>441</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="l_screen_video">
      <property name="text">
       <string>Video Source</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_screen_video">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_screen_compression">
      <property name="text">
       <string>Compression</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_screen_compression">
      <item>
       <property name="text">
        <string>Compressed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Uncompressed</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_screen_frame_rate">
      <property name="text">
       <string>Frame Rate</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_screen_frame_rate">
      <item>
       <property name="text">
        <string>60p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>59.95p (NTSC)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29.97p (NSTC)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25 (PAL)</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_screen_audio">
      <property name="text">
       <string>Audio Source</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_screen_audio">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb_stop">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>670</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_start">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>670</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_camera">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>210</y>
     <width>261</width>
     <height>441</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="l_camera_video">
      <property name="text">
       <string>Video Source</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_camera_video">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_camera_compression">
      <property name="text">
       <string>Compression</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_camera_compression">
      <item>
       <property name="text">
        <string>Compressed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Uncompressed</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_camera_frame_rate">
      <property name="text">
       <string>Frame Rate</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_camera_frame_rate">
      <item>
       <property name="text">
        <string>60p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>59.95p (NTSC)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29.97p (NSTC)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25 (PAL)</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_camera_audio">
      <property name="text">
       <string>Audio Source</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_camera_audio">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="l_logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>350</width>
     <height>136</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="NDI_SDK.qrc">:/resources/images/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_title">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 40px;
}</string>
   </property>
   <property name="text">
    <string>NDI Streamer</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_screen">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 32px;
}</string>
   </property>
   <property name="text">
    <string>SCREEN</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_camera">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 32px;
}</string>
   </property>
   <property name="text">
    <string>CAMERA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_close">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <zorder>pb_stop</zorder>
  <zorder>pb_start</zorder>
  <zorder>l_logo</zorder>
  <zorder>l_title</zorder>
  <zorder>gb_screen</zorder>
  <zorder>gb_camera</zorder>
  <zorder>l_screen</zorder>
  <zorder>l_camera</zorder>
  <zorder>pb_close</zorder>
 </widget>
 <resources>
  <include location="NDI_SDK.qrc"/>
 </resources>
 <connections/>
</ui>
